packages/opds-browser/src/components/__tests__/Book-test.tsx

```diff
@@ -20,38 +20,44 @@ let book: BookData = {
 
 describe("Book", () => {
   it("shows the book cover with empty alt", () => {
-    let renderedBook = TestUtils.renderIntoDocument(
+    let wrapper = shallow(
       <Book book={book} />
-    ) as Book;
+    );
 
-    let coverImage = TestUtils.findRenderedDOMComponentWithTag(renderedBook, "img");
-    expect(coverImage.getAttribute("src")).toEqual(book.imageUrl);
-    // alt is blank so screen readers don't read image filename
-    expect(coverImage.getAttribute("alt")).toEqual("");
+    let link = wrapper.find(BrowserLink);
+    let coverImage = link.children().at(0);
```

Comment:
Is it important to test that the cover image is the first child? I would do link.find("img") to make sure it shows up somewhere.